### DISCUSSION TITLE
IMAP subfolders

### DISCUSSION THREAD

**Discussion by robin-collins** [jonaswinkler/paperless-ng#1505] [Support]
Hello,

I am having no luck in getting my rules to check IMAP subfolders. I have read the 'hint' about dots, but I am not sure what is expected.

I have tried.
INBOX\subfolder
INBOX.subfolder
INBOX..subfolder
INBOX:subfolder

none appear to work.
INBOX on its own works fine.
Please help (and provide a clear example in your hint)

**Comment by stefanorth** [comment]
Hey @Robin-Collins,
I've struggled with subfolders yesterday too and found out, that it works for me when I use a slash to separate folder names. A working config is: `INBOX/Paperless/New`.
